File: linguagens/Java/exercicios/idade.java
import java.util.Scanner;

public class idade
{
	public static void main(String[] args) {

		double idade1, idade2, media;
		String nome1, nome2;

		Scanner sc = new Scanner(System.in);

		System.out.println("coloque o nome da primeira pessoa: ");

		nome1 = sc.nextLine();

		System.out.println("coloque a idade da primeira pessoa: ");

		idade1 = sc.nextDouble();

		System.out.println("coloque o nome da segunda pessoa: ");

		nome2 = sc.nextLine();

		System.out.println("coloque a idade da segunda pessoa: ");

		idade2 = sc.nextDouble();

		media = (idade1 + idade2) / 2;

		System.out.println("a media de idade de " + nome1 + " e "  + nome2 + " = "  + String.format("%.1f",media));
		
		sc.close();
	}
}

File: linguagens/C#/Exercicios/idade.cs
using System;
using System.Globalization;

namespace Programa {
class Program {
	static void Main(string[] args) {


		CultureInfo CI = CultureInfo.InvariantCulture;

		double idade1, idade2, media;
		string nome1, nome2;

		Console.WriteLine("Coloque o nome da pessoa 1: ") ;
		nome1 = Console.ReadLine();

		Console.WriteLine("Coloque a idade da pessoa 1: ");
		idade1 = double.Parse(Console.ReadLine(), CI);

		Console.WriteLine("Coloque o nome da pessoa 2: ") ;
		nome2 = Console.ReadLine();

		Console.WriteLine("Coloque a idade da pessoa 2: ");
		idade2 = double.Parse(Console.ReadLine(), CI);


		media = (idade1 + idade2)  / 2 ;

		Console.WriteLine("A idade media de " + nome1 + " e " + nome2  + " e de " + media.ToString("F2", CI));

	}
}
}